### PR TITLE
Update File Cache disk configuration settings default values

### PR DESCRIPTION
**Pull request by elenatorro**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

Update the docs to use the actual default values that are defined at https://github.com/ClickHouse/ClickHouse/blob/master/src/Interpreters/Cache/FileCache_fwd.h

### CODE REVIEW

**Review by kssenii — APPROVED** [review]

**Comment by elenatorro** [comment]
I'm also wondering if `bypass_cache_threashold` should be added to the docs as well, it seems to have been introduced here: https://github.com/ClickHouse/ClickHouse/pull/42418 🤔

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit 84872517708ad03c764e616fd28d5d1d12593697 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51680/84872517708ad03c764e616fd28d5d1d12593697/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by kssenii** [comment]
> I'm also wondering if bypass_cache_threashold should be added to the docs as well, it seems to have been introduced here: https://github.com/ClickHouse/ClickHouse/pull/42418 🤔

yes, good to add if you could, thanks
btw there was a typo in this setting, now it is renamed `threashold` -> `threshold`: 
https://github.com/ClickHouse/ClickHouse/blob/d1c7e13d08868cb04d3562dcced704dd577cb1df/src/Interpreters/Cache/FileCacheSettings.cpp#L41

**Comment by elenatorro** [comment]
Ok, thanks! let me add it then. I saw the typo as well but I'm assuming it's already being used as you pointed out.

File: docs/en/operations/storing-data.md [hunk 1]
```diff
@@ -184,13 +184,15 @@ These settings should be defined in the disk configuration section.
 
 - `enable_filesystem_query_cache_limit` - allow to limit the size of cache which is downloaded within each query (depends on user setting `max_query_cache_size`). Default: `false`.
 
-- `enable_cache_hits_threshold` - number which defines how many times some data needs to be read before it will be cached. Default: `0`, e.g. the data is cached at the first attempt to read it.
+- `enable_cache_hits_threshold` - number which defines how many times some data needs to be read before it will be cached, defined by `cache_hits_threshold`. Default: `0`, e.g. the data is cached at the first attempt to read it.
+
+- `enable_bypass_cache_with_threshold` - allows to skip cache completely in case the requested read range exceeds the threshold defined by `bypass_cache_threashold`. Default: `268435456` (`256Mi`).
```
Comment (kssenii) [inline docs/en/operations/storing-data.md:189]: let's mention that default for `enable_bypass_cache_with_threshold` is `false`

File: docs/en/operations/storing-data.md [hunk 2]
```diff
@@ -184,13 +184,15 @@ These settings should be defined in the disk configuration section.
 
 - `enable_filesystem_query_cache_limit` - allow to limit the size of cache which is downloaded within each query (depends on user setting `max_query_cache_size`). Default: `false`.
 
-- `enable_cache_hits_threshold` - number which defines how many times some data needs to be read before it will be cached. Default: `0`, e.g. the data is cached at the first attempt to read it.
+- `enable_cache_hits_threshold` - number which defines how many times some data needs to be read before it will be cached, defined by `cache_hits_threshold`. Default: `0`, e.g. the data is cached at the first attempt to read it.
```
Comment (kssenii) [inline docs/en/operations/storing-data.md:187]: same here

**Review by kssenii — CHANGES_REQUESTED** [review]

**Review by kssenii — APPROVED** [review]

**Pull request merged**